src/schema/v2/artwork/__tests__/artwork.test.js

```diff
@@ -411,6 +411,49 @@ describe("Artwork type", () => {
     })
   })
 
+  describe("#edition_size", () => {
```

Comment:
🐫 pls

Reply:
I was following the syntax of all the others tests here...but maybe its just a holdover from before v2?

Reply:
Yup likely a hold over! 

Here's my thinking about this kind of stuff: A general rule of thumb I tend to follow is that evolution is always ok -- a similar example is how we used to `snake_case.js` all of our file names way back when, but when we started writing more idiomatic JS we started `camelCasing.js` them -- however, most files in a given directory are still `snake_case`, and rather than renaming everything, we just create all new files in `camelCase` and accept that newer code is more important than old and that mixing some pattterns is ok within reason. 

Same applies here -- even tho old props from v1 were `snake_case` new props are camel, and in an effort to align new code and other find-and-replacable bits 1:1 with each other, its an acceptable trade off to ignore older patterns within a file.